src/UI/SvgIcon/SvgIcon.stories.tsx

```diff
@@ -1,16 +1,28 @@
 import React from "react";
-import { SvgIcon, SvgIconArrowLeft } from "./SvgIcon";
+import { SvgIcon, SvgIconArrowLeft, SvgIconPaperClip, SvgIconX } from "./SvgIcon";
 
 export default {
-  title: "SvgIconArrowLeft|SvgIconArrowLeft",
-  component: SvgIconArrowLeft,
+  title: "SvgIcons|SvgIcon",
```

Comment:
```suggestion
  title: "SvgIcons/SvgIcon",
```
think / instead of | is the more latest way of grouping storybook components

Reply:
ok will change it